.ci/azure-pipelines-build.yml

```diff
@@ -24,3 +24,53 @@ jobs:
 
       - script: 'yarn build'
         displayName: 'Build'
+
+  - job: BuildDocker
+    displayName: 'Build Docker'
+
+    strategy:
+      matrix:
+        amd64:
+          BuildConfiguration: amd64
+        arm64:
+          BuildConfiguration: arm64
+        armhf:
+          BuildConfiguration: armhf
+
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    variables:
+      - name: Version
+        value: 0.0.0
+
+    steps:
+      - script: echo "##vso[task.setvariable variable=Version]$( awk -F '/' '{ print $NF }' <<<'$(Build.SourceBranch)' | sed 's/^v//' )"
```

Comment:
Would something like this work? `$[replace(variables['Build.SourceBranch'],'refs/tags/v','')]`

Reply:
I have no idea, I took this line from [jellyfin-server](https://github.com/jellyfin/jellyfin/blob/2375c35c4a966a469dd97f33deb57b342498b905/.ci/azure-pipelines-package.yml#L119).

Reply:
Just because I touched the Java API CI today this is how that CI gets the version (note that would still result in vX.Y.Z)

```yaml
      - script: 'echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$(git describe --tags)"'
        displayName: 'Set Version Variable'
```

Reply:
> Just because I touched the Java API CI today this is how that CI gets the version (note that would still result in vX.Y.Z)
> 
> ```yaml
> - script: 'echo "##vso[task.setvariable variable=JELLYFIN_VERSION]$(git describe --tags)"'
>   displayName: 'Set Version Variable'
> ```
> 
> /disclaimer: I did not touch this piece of the CI

okay due to this comment I may have found the reason for some artifact upload errors in the Java API ci. (**POSSIBLY**)
So pleas disregard my earlier comment

Reply:
I mean it would just mean you don't need the extra step, you can assign the variable at the top of the file using that expression if required.

Reply:
Should be resolved with the latest commit if I understood you correctly.

Reply:
Yes looks good.